### DISCUSSION TITLE
Assistance with Quantifying Mitochondrial Surface Area using Axondeepseg

### DISCUSSION THREAD

**Discussion by LilianLotte** [neuropoly/axondeepseg#809] [Help needed]
Hi everyone,

I am writing to seek your guidance on a technical aspect of my current project in Dr. Anmar Khadra’s lab in the Department of Physiology at McGill University.

As part of my internship, I am working on quantifying the surface area of mitochondria within the axons of mice populations using Axondeepseg. Despite my efforts, I am encountering challenges in proceeding with this analysis. I would appreciate any guidance on the steps needed to effectively utilize Axondeepseg for this purpose.

Here is an example of an image that I need to analyze (see attached). The quality is not perfect because I couldn't upload more than 10MB.

![CST-x9300-1-min](https://github.com/axondeepseg/axondeepseg/assets/150194622/8a6f0077-919b-45e2-a2b3-3ecfbd10827d)

Thank you very much for your time and assistance.

Best regards,
Lilian LOTTE

**Comment by Stoyan-I-A** [comment]
Hello @LilianLotte,
We'll have to discuss the team how we can help. Currently, our models segment the Axon parts and the Myelin parts of images. But what you would need is a model that segments mitochondria.

**Reply by LilianLotte** [reply]
Hi @Stoyan-I-A ,

Thank you for your quick response. Do you have any idea of how long it might take to develop a new model for mitochondria segmentation ? Additionally, if there is anything I can do to help or facilitate the process, please let me know.

**Reply by jcohenadad** [reply]
If you were to produce a few manual segmentations of mitochondria we could try to train a model on our end. @hermancollin do you have some bandwidth? Or maybe we could ask Reda?

**Reply by LilianLotte** [reply]
That would be great. Do you know approximately how many mitochondria segmentations you would need to train the model ?

**Reply by hermancollin** [reply]
Hey @LilianLotte! If I understand correctly, you would like a model to segment the mitochondria inside the axons. This sounds pretty straightforward. To train a model dedicated to this task, I think ~50-100 segmented mitochondrion would provide a good accuracy. If you want to collaborate, we can setup a meeting (next week?) to discuss your project details and timeline. At a glance, I think this could realistically be done in 2-3 months.

**Reply by LilianLotte** [reply]
Hi, 
Thank you for the quick feedback, unfortunately, 2-3 months will be too long because my internship here in McGill only lasts until August 1st. I will try to find another alternative to realize this mitochondria segmentation.

**Comment by hermancollin** [comment]
For future reference, we had a discussion with Dr. Anmar Khadra and Nianqi Deng this morning. We'll collaborate with their group to train a segmentation model for mitochondria.
